.github/workflows/website.yml

```diff
@@ -10,59 +10,59 @@ jobs:
   build-website:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-      # we need the full history to be able to get the chemiscope version with
-      # git describe during the build
-      with:
-        fetch-depth: 0
-    - uses: actions/setup-node@v1
-      with:
-        node-version: '12.x'
-    - uses: actions/setup-python@v2
-      with:
-        python-version: '3.x'
-    - name: install npm dependencies
-      uses: bahmutov/npm-install@v1
-    - name: install python dependencies
-      run: |
-        python3 -m pip install -r docs/requirements.txt
-        python3 -m pip install beautifulsoup4 requests
-    - run: npm run build
-    - run: npm run download-example-input
-    - run: cd docs && make html
-    - run: python3 utils/generate_standalone.py -o chemiscope_standalone.html
-    - name: assemble full website
-      run: |
+      - uses: actions/checkout@v2
+        # we need the full history to be able to get the chemiscope version with
+        # git describe during the build
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - name: install npm dependencies
+        uses: bahmutov/npm-install@v1
+      - name: install python dependencies
+        run: |
+          python3 -m pip install -r docs/requirements.txt
+          python3 -m pip install beautifulsoup4 requests
+      - run: npm run build
+      - run: npm run download-example-input
+      - run: cd docs && make html
+      - run: python3 utils/generate_standalone.py -o chemiscope_standalone.html
+      - name: assemble full website
+        run: |
           cp -r app/ gh-pages/
           cp dist/*.min.js gh-pages/
           cp chemiscope_standalone.html gh-pages/
           cp -r docs/_build/html/ gh-pages/docs/
           rm -f gh-pages/.gitignore
           tar czf gh-pages.tar.gz gh-pages/
-    - name: store website archive
-      uses: actions/upload-artifact@v1
-      with:
+      - name: store website archive
+        uses: actions/upload-artifact@v1
+        with:
           name: website
           path: gh-pages.tar.gz
 
-    - name: deploy website to gh-pages
-      # only deploy website from master
-      if: github.ref == 'refs/heads/master'
-      uses: peaceiris/actions-gh-pages@v3
-      with:
-        github_token: ${{ secrets.GITHUB_TOKEN }}
-        publish_dir: ./gh-pages/
-        cname: chemiscope.org
-        force_orphan: true
+      - name: deploy website to gh-pages
+        # only deploy website from master
+        if: github.ref == 'refs/heads/master'
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./gh-pages/
+          cname: chemiscope.org
+          force_orphan: true
```

Comment:
Are these new indentations a function of the new linter? Just asking because it's hard to parse.

Reply:
not the linter, prettier. We could disable yaml formatting with prettier, what do you think?